### DISCUSSION TITLE
What is the "App Directory" for?

### DISCUSSION THREAD

**Discussion by alexander-schranz** [Haehnchen/idea-php-symfony2-plugin#2016] [Q&A] [answered]
In the Symfony Plugin it is possible to select a `App Directory` which is by default `app` but in the default Symfony Skeleton this kind of directory does not exist since Symfony 4.

What was before `app/cache` and `app/config` was split into `config` (`config/packages`, `config/routes`, ...) and `var/cache`. 

So I'm not sure what is the correct value to be configured for `App Directory` in the Plugin?

**Comment by Matt-PMCT** [comment, marked as answer]
The plugin has not updated its settings to match the new directory structure. I have an open issue for this #2078. In the meantime, and in case anyone else has this question: In Symfony 4,5,6, the "app directory" should be pointing to the "config" folder and the "web directory" should be pointing to the "public" folder.

**Comment by Haehnchen** [comment]
planing to remove the configuration, every feature should work out of the box, mainly its just still there to support some "old Symfony" features.
